test/Instances.hs

```diff
@@ -79,7 +81,7 @@ instance Ord Value where
   _        `compare` _        = GT
 #endif
 
-#if MIN_VERSION_aeson(2,0,0)
+#if !MIN_VERSION_aeson(2,0,3)
```

Comment:
This will not compile for anything `< aeson-2.0`, so this should be:
```
#if MIN_VERSION_aeson(2,0,0) && !MIN_VERSION_aeson(2,0,3)
```

Reply:
Thanks, fixed in f08a73b